client/app/components/Form/DynamicForm.jsx

```diff
@@ -8,63 +9,69 @@ import { REQUIRES_DEFAULT } from '../Input/InputDefault';
 import { Utils } from '../../utils';
 import css from './Form.scss';
 
-type KeyProps = { myKey?: any };
+type KeyProps = { myKey?: string };
 
 type MyInputProps = InputProps & KeyProps;
 
+type FormProps = {
+  action: string,
+  inputs: MyInputProps[],
+};
+
 type Errors = { [string]: boolean } | {};
 
 export type Props = {
+  // Somehow Flow does not detect nameValue being used in a function outside the component
```

Comment:
Oh no, that's strange :O

Reply:
Yes, strange. Even though when I've reverted this change and there wasn't any error in vs-code, circleci test failed on that line exactly :/